src/gui/widgets/listbox.cpp

```diff
@@ -1095,16 +1094,14 @@ widget* builder_horizontal_listbox::build() const
 	list_view* widget = new list_view(
 			true, true, generator_base::horizontal_list, true, list_builder);
 
-	init_control(widget);
+	//init_control(widget);
```

Comment:
Delete these rather than commenting them.

Also, I'm wondering where the work of these functions is done now? Presumably I've just missed it.